### ISSUE TITLE
Rule MD033 edge case with blockquotes

### ISSUE THREAD

**Issue by pfeif** [rvben/rumdl#105]
I'm not sure whether this is edge case is worth addressing, but I wanted to point it out anyway in case you decide rumdl should ignore quoted text/code.

If you use the blockquote character `>` to quote an HTML block with an anchor element, rumdl reports an error. I know that sentence isn't particularly clear, so below is an example where I'm directly quoting the [htmx user docs](https://htmx.org/docs/) as seen in this screenshot.

<img width="1014" height="125" alt="Image" src="https://github.com/user-attachments/assets/401897e3-d687-488c-a15f-e6bb5508de37" />

    > It's worth mentioning that, if you prefer, you can use the [`data-`](https://html.spec.whatwg.org/multipage/dom.html#attr-data-*) prefix when using htmx:
    >
    > ```html
    > <a data-hx-post="/click">Click Me!</a>
    > ```

This block reports a rule MD033 violation

```
.\README.md:32:3: [MD033] Inline HTML found: <a data-hx-post="/click"> (use Markdown syntax instead)
.\README.md:32:37: [MD033] Inline HTML found: </a> (use Markdown syntax instead)
```

However, if the blockquote is removed, it does not flag the html for the same violation.

    ```html
    <a data-hx-post="/click">Click Me!</a>
    ```

**Comment by rvben** [comment]
Hi @pfeif,

Thanks for raising the issue. I indeed had no proper support for inline code nested in block quotes. This is now added in 0.0.157.

Thanks again for this issue and making rumdl better. 👍
